contentcuration/contentcuration/frontend/shared/views/contentNodeFields/CategoryOptions.vue

```diff
@@ -0,0 +1,346 @@
+<template>
```

Comment:
Not necessary to go back and change, but sometimes nice to do a `git mv` when moving files from one location to another, as it helps retain the git blame. Same applies to the other files that have moved. Something to look at in future move arounds!

Reply:
Yes, git has got better at tracking when a file has moved without modification (and github has some extra tricks for catching this even when git itself does not) - but when there's a modification, it's always safest to do `git mv`, commit, then modify.